Power_Supervisor.lib

```diff
@@ -140,6 +140,50 @@ X VSS 3 -100 -400 100 U 50 50 1 1 W
 ENDDRAW
 ENDDEF
 #
+# MCP39F521
+#
+DEF MCP39F521 U 0 20 Y Y 1 F N
+F0 "U" -450 750 50 H V C CNN
+F1 "MCP39F521" 350 750 50 H V C CNN
+F2 "Package_DFN_QFN:QFN-28-1EP_5x5mm_P0.5mm_EP3.35x3.35mm" 0 100 50 H I C CNN
+F3 "" 0 100 50 H I C CNN
+$FPLIST
+ QFN*1EP*5x5mm*P0.5mm*EP3.35x3.35mm*
```

Comment:
I've seen other PRs recently omit the EP size from the filter, for robustness.
```suggestion
 QFN*1EP*5x5mm*P0.5mm*
```

Reply:
Agree